tests/git-scripts/README.md

```diff
@@ -6,6 +6,11 @@ for more information, see the [git documentation](https://git-scm.com/docs/githo
 
 The mbed TLS git hooks are located in `<mbed TLS root>/tests/git-scripts` directory, and one must create a soft link from `<mbed TLS root>/.git/hooks` to `<mbed TLS root>/tesst/git-scripts`, in order to make the hook scripts successfully work.
 
+Prerequisites for running the scripts:
+* python
+* perl
+* pylint (installed with `pip install pylint`)
```

Comment:
Does it require a specific python or pylint version? I remember there was a disagreement/issue with this at some point

Reply:
It does require some pylint version, not sure which though...
When I installed pylint through  aptitude (`apt-get install pylint`) it was an old version. This is why it should be installed through `pip`, which installs a later version